docs/pages/views.rb

```diff
@@ -130,6 +130,28 @@ def template(&content)
 					end
 					```
 				MD
+
+				render Markdown.new(<<~MD)
+					## Registering custom elements
+
+					You can register custom elements with the `register_element` macro. The custom element will only be available in the view where it is registered and subclasses of that view.
+				MD
+
+				render Example.new do |e|
+					e.tab "hello.rb", <<~RUBY
+            class Hello < Phlex::View
+              register_element :hello
```

Comment:
HTML custom elements must contain a dash, so let's include that in our example.

```suggestion
              register_element :trix_editor
```